ChatQnA/docker/xeon/README.md

```diff
@@ -57,6 +57,34 @@ chaqna-xeon-ui-server
 Port 5173 - Open to 0.0.0.0/0
 ```
 
+You can pull ChatQnA and components docker images form docker hub or build images from source code.
+
+## 🚀 Pull docker images
+
+```bash
+docker pull opea/dataprep-redis:latest
+docker pull opea/embedding-tei:latest
+docker pull opea/retriever-redis:latest
+docker pull opea/reranking-tei:latest
+docker pull opea/chatqna:latest
+docker pull opea/chatqna-ui:latest
+```
+
+### Use TGI as backend
+
+```
+docker pull opea/llm-tgi:latest
+```
+
+### Use vLLM as backend
+
+```
+docker pull opea/vllm:latest
+docker pull opea/llm-vllm:latest
+```
+
+Go to [Section Check Docker Images)](#Check-Docker-Images)
+
 ## 🚀 Build Docker Images
```

Comment:
This section should be moved to a backup section. Keep the flow simple for developer to get started.